kpops/component_handlers/helm_wrapper/helm.py

```diff
@@ -210,13 +210,17 @@ def load_manifest(yaml_contents: str) -> Iterator[HelmTemplate]:
     def __execute(self, command: list[str]) -> str:
         command = self.__set_global_flags(command)
         log.debug(f"Executing {' '.join(command)}")
-        process = subprocess.run(
-            command,
-            check=True,
-            capture_output=True,
-            text=True,
-        )
-        Helm.parse_helm_command_stderr_output(process.stderr)
+        try:
+            process = subprocess.run(
+                command,
+                check=True,
+                capture_output=True,
+                text=True,
+            )
+        except subprocess.CalledProcessError as e:
+            Helm.parse_helm_command_stderr_output(e.stderr)
+            log.debug(e.stdout)
```

Comment:
Do you think this is a debug log or an error one?

Reply:
Debug. I kept the behavior identical to the one prior to introducing the bug. see https://github.com/bakdata/kpops/pull/376#discussion_r1368230652